vnext/JSI/Universal/ChakraRtRuntime.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#ifndef CHAKRACORE
+
+#include "ChakraRuntime.h"
+#include "ChakraRuntimeArgs.h"
+
+#include <memory>
+#include <string>
+
+namespace Microsoft::JSI {
+
+class ChakraRtRuntime : public ChakraRuntime {
+ public:
+  using ChakraRuntime::ChakraRuntime;
+  ~ChakraRtRuntime() noexcept = default;
+
+#pragma region Functions_inherited_from_Runtime
```

Comment:
>#pragma region [](start = 0, length = 14)

nit: I think pragma region is useful for large spans of code, but it seems a bit overkill here. What about a comment that just says "Runtime" and "ChakraRuntime" above the block of functions?

Reply:
I was originally using comments in ChakraRuntime.h until the span got too large. That's when I switch to #pragma region and ended up copying those here. I'll change these to comments.

---
In reply to: [345415851](https://github.com/microsoft/react-native-windows/pull/3650#discussion_r345415851) [](ancestors = 345415851)